src/reference/asciidoc/whats-new.adoc

```diff
@@ -116,5 +116,5 @@ In addition, the synchronizers for inbound channel adapters can now be provided
 
 ==== Twitter Support
 
-Since Spring Social project is in the https://spring.io/blog/2018/07/03/spring-social-end-of-life-announcement[End of Life], the Twitter support in Spring Integration has been moved to the Extensions project.
+Since Spring Social project Since Spring Social project moved to https://spring.io/blog/2018/07/03/spring-social-end-of-life-announcement[End of Life Status], Twitter support in Spring Integration has been moved to the Extensions project.
```

Comment:
s/Since Spring Social project Since Spring Social project moved to/Since Spring Social project **has** moved to/